packages/tools/devtools/devtools-browser-extension/src/devtools/TelemetryLogging.ts

```diff
@@ -0,0 +1,119 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import * as UTel from "@microsoft/oteljs";
```

Comment:
I might have asked this before but why is this being imported as UTel? What's the significance between UTel vs OTel?

Reply:
Good question! I took it from an example, but I believe the reasoning is that the packages are named OTel because they're meant to enable apps to use the OpenTelemetry standard. Then the internal SDKs built on top of them go by the name _Unified_ Telemetry, so UTel. But really we could import with any name we want.